### PR TITLE
Change some sentences for better clarity

### PR DESCRIPTION
**Pull request by 1407arjun**
## Summary
Issue jquery#4928
Made the following changes in the README.md file:

1. Added a git clone code below the clone instruction.
2. Changed some sentences to sound better.
3. Added an article to the heading tagline.

## Checklist
- [ ] All authors have signed the CLA at https://cla.js.foundation/jquery/jquery
- [ ] New tests have been added to show the fix or feature works
- [ ] Grunt build and unit tests pass locally with these changes
- [ ] If needed, a docs issue/PR was created at https://github.com/jquery/api.jquery.com

Not really sure about the checklist as I'm creating a PR for the first time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[jQuery](https://jquery.com/) — New Wave JavaScript
```
Comment (mgol) [inline README.md:1]: That slogan has been there for 15 years; I'm not sure there's a need to change it now.

File: README.md [hunk 2]
```diff
@@ -15,17 +15,17 @@ In the spirit of open source software development, jQuery always encourages comm
 3. [Writing Code for jQuery Foundation Projects](https://contribute.jquery.org/code/)
 
 
-Environments in which to use jQuery
+Environments which support jQuery
```
Comment (mgol) [inline README.md:18]: This is not correct. Those environments are not doing anything specifically to support jQuery, it's jQuery that make sure it works fine in those environments.

File: README.md [hunk 3]
```diff
@@ -15,17 +15,17 @@ In the spirit of open source software development, jQuery always encourages comm
...
 - [Browser support](https://jquery.com/browser-support/)
-- jQuery also supports Node, browser extensions, and other non-browser environments.
+- jQuery is also supported by Node.js, browser extensions, and other non-browser environments.
 
 
 What you need to build your own jQuery
 --------------------------------------
 
-To build jQuery, you need to have the latest Node.js/npm and git 1.7 or later. Earlier versions might work, but are not supported.
+To build your own jQuery, you need to have the latest Node.js/npm and git 1.7 or later. Earlier versions might work, but are not supported.
```
Comment (mgol) [inline README.md:28]: Why "your own"? I don't see an issue with the previous text.

File: README.md [hunk 4]
```diff
@@ -40,6 +40,9 @@ How to build your own jQuery
 ----------------------------
 
 First, [clone the jQuery git repo](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository).
+```bash 
+git clone https://github.com/jquery/jquery.git
+```
```
Comment (mgol) [inline README.md:45]: We've had a concrete instruction in the past but we've deleted that on purpose to not maintain that piece of the documentation considering how many ways there are to clone the repo. The link above should suffice.

File: README.md [hunk 5]
```diff
@@ -73,7 +76,7 @@ Special builds can be created that exclude subsets of jQuery functionality.
 This allows for smaller custom builds when the builder is certain that those parts of jQuery are not being used.
 For example, an app that only used JSONP for `$.ajax()` and did not need to calculate offsets or positions of elements could exclude the offset and ajax/xhr modules.
 
-Any module may be excluded except for `core`, and `selector`. To exclude a module, pass its path relative to the `src` folder (without the `.js` extension).
+Any module may be excluded except for the `core`, and `selector` modules. To exclude a module, pass its path relative to the `src` folder (without the `.js` extension).
```
Comment (mgol) [inline README.md:79]: `core` is the name here, why do we need the article? We're not writing "the Firefox". "except for" also suggests what's mentioned are modules.

File: README.md [hunk 6]
```diff
@@ -113,7 +116,7 @@ grunt custom --amd=""
 
 #### Custom Build Examples
 
-To create a custom build, first check out the version:
+To create a custom build, first check out the build version:
```
Comment (mgol) [inline README.md:119]: this is not a version of the build but version of jQuery; we're checking out a tag in the repository.

**Review by mgol — CHANGES_REQUESTED** [review]
Thanks for the contribution but I feel changes in this PR are either not necessary or even wrong so we shouldn't land it.

**Comment by goarblt** [comment]
hello
https://www.evil.com

**Comment by goarblt** [comment]
hello
https://www.evil.com

**Pull request closed without merging**